tests/lib/entropy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+feed_kernel_entropy_pool() {
+	# Feed the kernel entropy pool when the level is under 700 (level used by ubuntu-core)
+	if [ $# -gt 0 ] && [ "$1" = "--force" ] || [ $(sysctl --values kernel.random.entropy_avail || echo 0) -gt 700 ]; then
+		echo "HRNGDEVICE=/dev/urandom" > /etc/default/rng-tools
+		/etc/init.d/rng-tools restart
```

Comment:
I'm merging this as a step forward, but can't we do this by default in all cases during the project prepare?  We don't care about the entropy generated by those systems. Nothing we'll generate there will be used for anything but testing.